tapis-project/tapis-ui
# task/TUI-190 - system information

## Overview:

Replace files view in system detail with SystemDetail view

## Related Github Issues:

- [TUI-190](https://github.com/tapis-project/tapis-ui/issues/190)

## Summary of Changes:

- Added details and useDetails to tapis-api and tapis-hooks
- Added SystemDetail tapis-ui component
- Added SystemDetail layout in tapis-app, to replace FileListing

## Testing Steps:

1. npm install
2. npm start
3. Log in as service account
4. Click on a system to see its details

## UI Photos:

## Notes:
